From: Dridi Boukelmoune <dridi.boukelmoune@gmail.com>
Date: Wed, 17 Oct 2018 12:42:22 +0200
Subject: 32bit-friendly format in vmod-tcp

Fixes #122
---
 src/vmod_tcp.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/vmod_tcp.c b/src/vmod_tcp.c
index fee4bd3..570be45 100644
--- a/src/vmod_tcp.c
+++ b/src/vmod_tcp.c
@@ -174,7 +174,7 @@ vmod_set_socket_pace(VRT_CTX, VCL_INT rate)
             sizeof(pacerate)) != 0)
                 VSLb(ctx->vsl, SLT_VCL_Error, "set_socket_pace(): Error setting pace rate.");
 	else
-                VSLb(ctx->vsl, SLT_VCL_Log, "vmod-tcp: Socket paced to %lu KB/s.", rate);
+                VSLb(ctx->vsl, SLT_VCL_Log, "vmod-tcp: Socket paced to %jd KB/s.", (intmax_t)rate);
 
 #  ifndef NDEBUG
         int retval;
